ee/vellum_ee/workflows/display/nodes/vellum/error_node.py

```diff
@@ -21,6 +22,12 @@ def serialize(
         node_id = self.node_id
         error_source_input_id = self.node_input_ids_by_name.get("error_source_input_id")
 
+        error_attribute = raise_if_descriptor(self._node.error)
+        input_values_by_name = {
+            "error_source_input_id": error_attribute,
```

Comment:
This `"error_source_input_id"` mismatches the attribute name `"error"`. These kinds of differences will make base node cutover slightly more difficult when we start embarking on that, but manageable.